scripts/ci-e2e.sh

```diff
@@ -38,28 +34,44 @@ ARTIFACTS="${ARTIFACTS:-${PWD}/_artifacts}"
 mkdir -p "${ARTIFACTS}/logs/"
 
 ARCH=$(uname -m)
+OS=$(uname -s)
+IBMCLOUD_CLI_VERSION=${IBMCLOUD_CLI_VERSION:-"2.11.1"}
 PVSADM_VERSION=${PVSADM_VERSION:-"v0.1.7"}
 E2E_FLAVOR=${E2E_FLAVOR:-}
 REGION=${REGION:-"us-south"}
 
+[ "${ARCH}" == "x86_64" ] && ARCH="amd64"
+
 trap cleanup EXIT
 
 cleanup(){
     # stop the boskos heartbeat
     [[ -z ${HEART_BEAT_PID:-} ]] || kill -9 "${HEART_BEAT_PID}" || true
 }
 
+# Installing binaries from github releases
 install_pvsadm(){
-    [ "${ARCH}" == "x86_64" ] && ARCH="amd64"
+    platform="$(echo ${OS} | tr '[:upper:]' '[:lower:]')-${ARCH}"
 
-    # Installing binaries from github releases
-    curl -fsL https://github.com/ppc64le-cloud/pvsadm/releases/download/${PVSADM_VERSION}/pvsadm-linux-${ARCH} -o pvsadm
+    curl -fsL https://github.com/ppc64le-cloud/pvsadm/releases/download/${PVSADM_VERSION}/pvsadm-${platform} -o pvsadm
     chmod +x ./pvsadm
 }
 
+install_ibmcloud_cli(){
+    if [ ${OS} == "Linux" ]; then
+        platform="linux-${ARCH}"
+    elif [ ${OS} == "Darwin" ]; then
+        platform="macos"
+    fi
+     
+    wget https://download.clis.cloud.ibm.com/ibm-cloud-cli/${IBMCLOUD_CLI_VERSION}/binaries/IBM_Cloud_CLI_${IBMCLOUD_CLI_VERSION}_${platform}.tgz
```

Comment:
wget will not be present by default in most of the containers, wondering if we can use curl instead.